src/docs/sdk/event-payloads/properties/transaction_info.mdx

```diff
@@ -0,0 +1,28 @@
+`transaction_info`
+
+: _Required_. Additional information about the name of the transaction.
+
+```json
+{
+  "transaction_info": {
+    "source": "raw-url"
+  }
+}
+```
+
+`transaction_info.source`
+
+: _Required_. Contains information about how the name of the transaction was determined.
+This will be used by the server to decide whether or not to scrub identifiers
+from the transaction name, or replace the entire name with a placeholder.
+
+| Source             | Description                                                                                                                     | <div style="width:220px">Examples</div>                |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| `raw-url`          | Raw URL, potentially containing identifiers.                                                                                    | `/auth/login/john123/`<br />`GET /auth/login/john123/` |
+| `raw-url-fallback` | The SDK is configured to send `url-pattern`, but has to fall back to a raw URL for this particular transaction.                 |
```

Comment:
I figured that for data analysis, it makes sense to distinguish between the case where the SDK is configured to send the raw URL, and the case where it is configured to send a parametrized URL but has to fall back to a raw URL instead. @AbhiPrasad what do you think, can these two cases even be distinguished in the JS SDK?

Reply:
I agree that we probably do not need to distinguish these cases. The actual source of this transaction name is the URL, and it's irrelevant whether it was applied as a fallback, as part of a multi-step process, or directly.

Reply:
It's feasible to distinguish between them, but I think it's also unnecessary (and a waste of bundle size) - let's stick with just the `url` source!

Reply:
Alright, removing url-fallback then.